src/rt/util/hash.d

```diff
@@ -17,22 +17,22 @@ version( AnyX86 )
     version = HasUnalignedOps;
 
 
-@trusted pure nothrow
+@trusted pure nothrow @nogc
 size_t hashOf( const(void)[] buf, size_t seed = 0 )
 {
     return hashOf(buf.ptr, buf.length, seed);
 }
 
-@trusted pure nothrow
-size_t hashOf( const(void)* buf, size_t len, size_t seed = 0 )
+@trusted pure nothrow @nogc
```

Comment:
Anyone can import it - it's in `import/core/internal/hash.d`

Changing that is properly a separate PR.